requirements/index.html

```diff
@@ -36,6 +36,38 @@
 
   <section id="introduction">
     <h2>Introduction</h2>
+    <p>The <a href="https://www.w3.org/WoT/">Web of Things (WoT)</a> seeks to counter the fragmentation of the
+      <a href="https://en.wikipedia.org/wiki/Internet_of_things">Internet of Things (IoT)</a> by using and extending
+      existing, standardized web technologies.
+    </p>
+    <p>The <a href="https://www.w3.org/TR/wot-thing-description/">W3C Web of Things (WoT) Thing Description 1.1
+        specification</a> [[wot-thing-description11]] defines an information model and JSON-based representation format
+      for describing the capabilities of connected devices and the interfaces with which to communicate with them.
+      Thing Descriptions are designed to be protocol-agnostic and flexible enough to describe a wide range of existing
+      IoT devices.</p>
+    <p>In order to provide this level of flexibility the Thing Description specification includes a number of extension
+      points including protocol bindings, payload bindings, security mechanisms, link relations and semantic contexts.
```

Comment:
```suggestion
    <p>To provide this level of flexibility, the Thing Description specification includes a number of extension
      points including protocol bindings, payload bindings, security mechanisms, link relations, and semantic contexts.
```

Reply:
@TallTed What's wrong with "In order to"?

We're going to have to agree to disagree about the Oxford comma 😉

Reply:
I strongly advise the terminal comma, as it has never introduced confusion in my experience, while omitting it has led to confusion many times (and I can see the potential here).

"In order to" is usually an unnecessary formalism where "to" usually communicates the same meaning. I won't lie down in the road on this one (though I might on other instances).